### DISCUSSION TITLE
Velero Slack channel access

### DISCUSSION THREAD

**Discussion by vijaynetha** [vmware-tanzu/velero#4212] [General discussions]
Hi team,

i need access to velero Slack channel.

**Comment by zubron** [comment]
HI @vijaynetha - The Velero Slack channel is on the Kubernetes Slack workspace. You can join that workspace here: https://slack.k8s.io/
Once you've signed up, you can access the channel here: https://kubernetes.slack.com/archives/C6VCGP4MT

**Comment by vijaynetha** [comment]
Thank Q.
